en/Getting started/Link notes.md

```diff
@@ -45,8 +45,9 @@ Another way to navigate between notes is through _backlinks_. A backlink lets yo
 Another way to navigate between your notes is by using a visual representation of how your notes are connected.
 
 1. In the upper-right corner of the note, click **More options** (three dots).
-2. Select **Open local graph**.
-3. Click any of the nodes in the graph to navigate to that note.
+2. Select **Open linked view**.
+3. Select **Open local graph**.
+4. Click any of the nodes in the graph to navigate to that note.
```

Comment:
Confirmed this is appropriate text for desktop application versions. Checking mobile.